L/Libxc/Libxc_GPU/build_tarballs.jl

```diff
@@ -5,61 +5,53 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "Libxc_GPU"
-version = v"6.1.0"
+version = v"7.0.0"
 include("../sources.jl")
 
-sources = [
-    sources;
-    DirectorySource("./bundled")
-]
-
 # Bash recipe for building GPU version
 # Notes:
 #   - 3rd and 4th derivatives (KXC, LXC) not built since gives a binary size of ~200MB
 script = raw"""
 cd $WORKSPACE/srcdir/libxc-*/
 
-# Needed for Libxc 6.1.0 as these backport some fixes on libxc master
-# On Libxc > 6.1.0 we can also remove the -DBUILD_TESTING=OFF
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/cmake-cuda.patch
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/source-fixes.patch
+ln -s $prefix/cuda/lib $prefix/cuda/lib64
 
 mkdir libxc_build
 cd libxc_build
-cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}\
     -DCMAKE_BUILD_TYPE=Release -DENABLE_XHOST=OFF -DBUILD_SHARED_LIBS=ON \
-    -DENABLE_CUDA=ON -DCMAKE_CUDA_COMPILER=$prefix/cuda/bin/nvcc -DBUILD_TESTING=OFF \
-    -DENABLE_FORTRAN=OFF -DDISABLE_KXC=ON ..
+    -DENABLE_CUDA=ON -DCMAKE_CUDA_COMPILER=$prefix/cuda/bin/nvcc \
```

Comment:
libxc is using [`enable_language(CUDA)`](https://gitlab.com/libxc/libxc/-/blob/devel/CMakeLists.txt?ref_type=heads#L17), i.e. the [FindCUDAToolkit](https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html) CMake module, so it should be sufficient (and a bit more proper) to specify the CUDA toolkit root, i.e. `CUDAToolkit_ROOT=$prefix/cuda` - instead of specifying the path to `nvcc`.

... but for unknown reasons, it seems `CUDA_TOOLKIT_ROOT_DIR=$prefix/cuda` is usually what actually works - though that is the variable specified for the deprecated [FindCUDA](https://cmake.org/cmake/help/latest/module/FindCUDA.html)

Reply:
None of the `CUDA_TOOLKIT_ROOT_DIR` or `CUDAToolkit_ROOT` option works out of the box in this build, without exporting the location of the `nvcc` compiler. I left the `DCMAKE_CUDA_COMPILER` as it is.

Reply:
Did you try setting CUDA_PATH?